README.md

```diff
@@ -310,6 +361,55 @@ function assignId (req, res, next) {
 }
 ```
 
+### arguments to custom token formats
+
+Example of a custom token format that uses an argument. It will output the
+property from `process` related to the argument passed to the token. If that property
+is a function then it will return the value returned from that function.
+
+```js
+var express = require('express')
+var morgan = require('morgan')
+
+morgan.token('process', function getId (req, res, arg) {
+  if (typeof process[arg] === 'function') {
+    return String(process[arg]())
+  } else if (process[arg]) {
+    return String(process[arg])
+  }
+})
+```
+
+### output JSON logs
+
+``` js
+var express = require('express')
+var morgan = require('morgan')
+
+var route = morgan.compile(':method :url')
+var outputJson = function (tokens, req, res) {
+  return JSON.stringify({
+    route: route(tokens, req, res),
+    status: morgan.status(req, res),
+    'content-length': morgan.res(req, res, 'content-length')
+  })
+}
+
+var app = express()
+
+app.use(morgan(outputJson))
+
+app.get('/', function (req, res) {
+  res.send('hello, world!')
+})
+```
+
+A request to `GET /` would cause `morgan` to log the following JSON:
+
+``` json
+{ "route": "GET /", "status": 200, "content-length": 13 }
```

Comment:
Sure. Changed this in [1c9416c](https://github.com/expressjs/morgan/pull/125/commits/1c9416c7692b359b2f4bcfe7b9b357fe14dc9306). Not sure why this Github comment wasn't hidden automatically because the line was changed.